tools/jbrowse2/jbrowse2.py

```diff
@@ -615,8 +615,10 @@ def write_config(self):
 
     def text_index(self):
         # Index tracks
-        args = ["jbrowse", "text-index"]
-        self.subprocess_check_call(args)
+        e = os.environ
+        e["SHELL"] = "/bin/sh"
+        cmd = ["jbrowse", "text-index"]
+        subprocess.run(cmd, env=e, shell=True)
```

Comment:
Unless the environment has that variable added, there is a problem with node.js trying to read the password file, remember? This one allows text indexing to be turned back on.
This is a fourth place where that could happen so am covering it.  We discussed changing the biocontainer env to avoid the need for this but that is an even worse option.
If you really want, this one could be reverted but the other three cases at line 760, 1114 and 1542 will result in failures.

Reply:
I don't remember at all ;) but now that you say it :)

Can you maybe add a comment to this part of the code? Feel free to merge after that.